samples/bootloader/boards/nrf54l15dk_nrf54l15_cpuapp.conf

```diff
@@ -4,3 +4,12 @@ CONFIG_ONOFF=n
 
 CONFIG_BOOT_BANNER=n
 CONFIG_NCS_BOOT_BANNER=n
+
+# Not uses by default ED25519 signature on nRF54l15
+CONFIG_PSA_USE_CRACEN_AEAD_DRIVER=n
+CONFIG_PSA_USE_CRACEN_KEY_MANAGEMENT_DRIVER=n
+# Hardware support for ED25519
+CONFIG_PSA_USE_CRACEN_HASH_DRIVER=y
+CONFIG_PSA_USE_CRACEN_MAC_DRIVER=y
```

Comment:
actually, can you rename this file `nrf54l15_cpuapp.conf` and move it from `boards` folder to `socs` folder, that way can work on all nrf54l15 socs including customer ones